frontend/src/common/constants/constants.ts

```diff
@@ -11,6 +14,7 @@ export const TEN_MINUTES = 10 * ONE_MINUTE;
 // Hour
 export const ONE_HOUR = 6 * TEN_MINUTES;
 
-// Contacts
+export const TOLL_FREE_HAVE_QUESTIONS = "Have questions? Please contact the Provincial Permit Centre. Toll-free:";
 export const TOLL_FREE_NUMBER = "1-800-559-9688";
 export const PPC_EMAIL = "ppcpermit@gov.bc.ca";
+export const PAY_NOW = "Pay Now";
```

Comment:
Um for now I think it's ok to keep the "Pay Now" in the PermitPayFeeSummary component as is, because if we do this we would probably have to take care of all texts throughout the codebase and place them in this constants file (which would become huge).

Reply:
+1, I am not sure I like the idea of having one constant file with hundreds of text descriptions. I like the way we use the text as is. This is not an optimization I'm keen on.

Reply:
no worries - updated